### DISCUSSION TITLE
Add pr approval button

### DISCUSSION THREAD

**Discussion by lavigne958** [community/community#4081] [Mobile]
Using the mobile for github I would like to be able to approve a PR.

**Comment by aconsuegra** [comment]
Hi @lavigne958! Thanks for getting in touch, it is possible to approve and merge PRs on both our Android and iOS apps. Could you let us know what issues are you seeing?

**Reply by lavigne958** [reply]
Oh ok, This is reassuring, I wondered why this was not integrated already.
Now you mention it I digged around and found that you can click on the mention I attach as screenshot, never though I could click on it, then I found the approve button. It's all good thank.
![gh_ui](https://user-images.githubusercontent.com/15084254/121051279-a0779200-c7b9-11eb-8dfc-167f36de8267.jpeg)

**Reply by aconsuegra** [reply]
We'll work on improving the user experience. Thanks for the feedback!
